### PR TITLE
CB-7-Routing

### PR DESCRIPTION
**Pull request by alekhyapinnamaneni**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
This PR adds routing to different pages of the Recipe Maker application.

* **What is the current behavior? What does the code do before you made the new changes?** 
Before the new changes, the user was stuck on the Home page and could not navigate to the Find a Recipe page.

* **What is the new behavior?**
Now the user can switch between the Home page and the Find a Recipe page.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No

* **Other information**:
n/a

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,13 +1,41 @@
...
   return (
     <>
       <MyNavbar/>
-      <LandingPage/>
+      <Router>
+          <Switch>
+            <Route path="/" exact component={LandingPage} />
+            <Route path="/test" component={Recipes} />
+          </Switch>
+      </Router>
```
Comment (AdemOdza) [inline frontend/src/App.js]: I think /test could be changed to something more relevant, both here and in MyNavbar. Maybe something like /FindRecipe?

**Review by AdemOdza — COMMENTED** [review]
Looks good! Just had the one comment about the page name.

**Review by sunghyun96 — APPROVED** [review]
I think other than the comment that Adem has already made, this looks good! I am wondering if there is a way to separate the jumbotron background color for the explanation component with the tools component so that the css styling will apply independently to each. I haven't found a way yet, but I think we can come back to that if there is time to do so later on. Nice job!

**Review by Stacey-Masi — APPROVED** [review]
Looks great!

File: frontend/src/App.js (hunk 1 above)
Comment (alekhyapinnamaneni) [reply frontend/src/App.js]: Thanks for the feedback! I changed /test to /findRecipe in App.js and in MyNavbar.js.

**Pull request merged**